src/xoreos.cpp

```diff
@@ -28,6 +28,7 @@
 #include <cstdio>
 
 #include <vector>
+#include <src/sound/fevfile.h>
```

Comment:
(The include format is wrong here...)

But this whole block is testing code you apparently accidentally left in :P